nativeauthenticator/nativeauthenticator.py

```diff
@@ -149,7 +150,11 @@ def get_or_create_user(self, username, pw, **kwargs):
             infos.update({'is_authorized': True})
 
         user_info = UserInfo(**infos)
-        self.db.add(user_info)
+        user = User(name=username)
+
+        self.db.add_all([user_info, user])
```

Comment:
where was the entry in the user table being created before?

Reply:
it wasn't hehehehe this is a fix so we can make the `delete_user` to work 😅